subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AsciidoctorPluginSmokeTest.groovy

```diff
@@ -50,6 +50,7 @@ class AsciidoctorPluginSmokeTest extends AbstractPluginValidatingSmokeTest {
                 "This is scheduled to be removed in Gradle 8.0. " +
                 "Please use the mainClass property instead. Consult the upgrading guide for further information: https://docs.gradle.org/${GradleVersion.current().version}/userguide/upgrading_version_7.html#java_exec_properties",
                 "https://github.com/asciidoctor/asciidoctor-gradle-plugin/issues/602")
+            .expectDeprecationWarning(deprecationOfFileTreeForEmptySources("sourceFileTree"), "TODO wolfs")
```

Comment:
What's the TODO about here?

Reply:
The TODO is about opening an issue in the asciidoctor Gradle plugin repository.

Reply:
I created an issue and added the link: https://github.com/asciidoctor/asciidoctor-gradle-plugin/issues/629